src/test/java/com/thealgorithms/misc/RangeInSortedArrayTest.java

```diff
@@ -0,0 +1,70 @@
+package com.thealgorithms.misc;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class RangeInSortedArrayTest {
+
+    @Test
+    public void testSortedRangeWithMultipleOccurrences() {
```

Comment:
Refactor these tests to parametrized please

Reply:
Done!